MTFLOW_caller.py

```
@@ -185,244 +179,149 @@ def __init__(self,
-                    file_handler.fileHandlingMTFLO(case_name=self.analysis_name,
-                                                   ref_length=self.ref_length).GenerateMTFLOInput(blading_params=self.blading_parameters,
-                                                                                                  design_params=self.design_parameters)  # Create the MTFLO input file
-                
-                MTFLO_call(self.analysis_name).caller() #Load in the blade row(s) from MTFLO
-
-                if debug:
-                    logger.info("Executing MTSOL")
+            if exit_flag != ExitFlag.CRASH:   
+                try:    
+                    MTFLO_call(self.analysis_name).caller() #Load in the blade row(s) from MTFLO
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)